tests/test_manager.py

```diff
@@ -63,30 +63,31 @@ def test_init_app(self):
         :class:`flask.ext.restless.APIManager` object.
 
         """
-        manager = APIManager()
-        manager.init_app(self.flaskapp, session=self.session)
-        manager.create_api(self.Person, app=self.flaskapp)
+        manager = APIManager(session=self.session)
+        manager.create_api(self.Person)
+        manager.init_app(self.flaskapp)
         response = self.app.get('/api/person')
         assert response.status_code == 200
 
     def test_init_app_split_initialization(self):
         manager = APIManager(session=self.session)
+        manager.create_api(self.Person)
         manager.init_app(self.flaskapp)
-        manager.create_api(self.Person, app=self.flaskapp)
         response = self.app.get('/api/person')
         assert response.status_code == 200
 
     def test_init_multiple(self):
-        manager = APIManager(session=self.session)
```

Comment:
Here also I would like to maintain the functionality that you are removing: a single manager should be able to create an API for two different Flask applications. This is a key part of being a Flask extension; see the [Flask extension development documentation][1] for more information.

[1]: http://flask.pocoo.org/docs/0.10/extensiondev/#the-extension-code